h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -102,7 +102,27 @@ def prostate_automl():
     print("Check predictions")
     print(aml.predict(train))
 
+    print("Check nfolds is passed through to base models")
+    aml = H2OAutoML(project_name="aml_nfolds3", nfolds=3, max_models=3, seed=1)
+    aml.train(y="CAPSULE", training_frame=train)
+    # grab the last model in the leaderboard, hoping that it's not an SE model
+    amodel = h2o.get_model(aml.leaderboard[aml.leaderboard.nrows-1,0])
+    # if you get a stacked ensemble, take the second to last 
+    # right now, if the last is SE, then second to last must be non-SE, but when we add multiple SEs, this will need to be updated
+    if type(amodel) == h2o.estimators.stackedensemble.H2OStackedEnsembleEstimator
+      amodel = h2o.get_model(aml.leaderboard[aml.leaderboard.nrows-2,0])
+    assert amodel.params['nfolds']['actual'] == 3
+
+    print("Check nfolds = 0 works properly")
+    aml = H2OAutoML(project_name="aml_nfolds0", nfolds=0, max_models=3, seed=1)
+    aml.train(y="CAPSULE", training_frame=train)
+    # grab the last model in the leaderboard (which will not be an SE model) and verify that nfolds = 0
```

Comment:
Is this always guaranteed?

Reply:
Nit: _if nfolds is not 5:_ is not necessary; the type check is always valid to make.

Reply:
@navdeep-G It's guaranteed that there's no ensembles because no SEs can be trained if `nfolds = 0` as long as it was passed through correctly.  So I'm checking that `nfolds = 0` is correctly passed through.